tests/unit_tests/testsettingsholder.cpp

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "testsettingsholder.h"
+
+#define SETTING(type, toType, getter, setter, remover, has, key, defaultValue, \
```

Comment:
is the use of `\` necessary here? I looked at a few other files in the same directory and not seeing the same pattern (ex: testenv.cpp, testaddonindex.cpp, testchckedint.cpp)

Reply:
That is because this is macro.